search/includes/classes/class-settingshealthjob.php

```diff
@@ -263,27 +262,69 @@ public function maybe_schedule_build_new_index( $indexable ) {
 	public function build_new_index( $indexable_slug ) {
 		update_option( self::BUILD_LOCK_NAME, time() ); // Set lock for starting rebuild.
```

Comment:
Do we need that option to be autoloaded?

Reply:
no